app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -94,6 +97,12 @@ def scoped_search(query, default_sort_by, default_sort_order, resource = resourc
 
       query = query.order("#{sort_attr} #{params[:sort_order] || default_sort_order}")
 
+      if includes.is_a?(Proc)
```

Comment:
also the example of an array:  https://github.com/Katello/katello/pull/5112/files#diff-f408c7192fcc44e7c0f5cb68ef90b82aR41